### PR TITLE
Update demos tutorials page

### PR DESCRIPTION
**Pull request by tushar-macrometa**

### CODE REVIEW

File: docs/demos/cf-lbrands.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# LBrands
+
+Macrometa-lbrands Forever 42 App is a full-stack e-commerce web application that creates a storefront (and backend) for customers to shop for fictitious fashion items. You can `browse` and `search` for books, look at `recommendations` and `best sellers`, `manage` your cart, `checkout`, `view` your orders, and more.
+
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/cf-lbrands.md:7]: @tushar-macrometa please take care of the formatting

File: docs/demos/fashion-store.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Fashion Store
+
+Fashion store Demo is a full-stack sample web application that creates a storefront (and backend) for customers to shop for clothes. You can `browse` and `search` for clothes, look at `recommendations` and `best sellers`, `manage` your cart, `checkout`, `view` your orders, and more.
+
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/fashion-store.md:7]: please take care of the formatting

File: docs/demos/fashion-store.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+|----------- |----------|-----------|--------------|-----------|
+| fashionstore@macrometa.io | `_system` | `xxxxxxxxxx` | [**Fashion Store App**](http://fashionstore.demo.macrometa.io/) ||
+
+### On GDN (Dynamo Mode)
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/fashion-store.md:13]: please take care of the formatting

File: docs/demos/fleet-management.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Fleet Management
+
+Fleet management demo demonstrates how you can monitor and take actions to keep a fleet running efficiently, and within budget using Macrometa GDN. With the app, you can monitor which `vehicles need maintenance`, what were the `unplanned maintenance cost`, and `overall fleet health`, etc.
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/fleet-management.md:6]: please take care of the formatting

File: docs/demos/multiplayer-fps.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Multiplayer FPS
+
+In this demo we are using FPS Unity game in which we are replacing the Network Layer to use Macrometa as the network layer. The fps game sent out the data to a stream which is processed/analysised by the stream workers to show various live statistics (like rtt,fps,live health,live shots fired, live grenades launched of/by the player, movement of the player in the game) on the dashboard for the end user to see. The dashboard also enables the cpability to record a live game and later replay the same whenever wanted.Also we have used ttl index for the lobbies feature in order to expire the outdated events.
+
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/multiplayer-fps.md:7]: please take care of the formatting

File: docs/demos/realtime-data-distribution.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Realtime Data Distribution (EventHub)
+
+Realtime Data Distribution app demos an event hub for real-time ingestion, analytics, and distribution of events using Macormeta GDN. With the app, the event publisher can publish the event into a GDN stream, and using the GDN stream processer you can generate analytics based on real-time data. In the demo, you can view analytics such as `Top securities sentiment`, `Securities sentiment`, and alongside all the real-time events.
+
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/realtime-data-distribution.md:7]: please take care of the formatting

File: docs/demos/realtime-stream-etl.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Realtime Stream ETL
+
+Building real-time ETL solution for synthetic bank data using GDN.
+
+### On GDN
+
+| **Tenant** | **Fabric** | **Password** | **GUI** | **Source Code**|
```
Comment (dwivedianurag) [inline docs/demos/realtime-stream-etl.md:7]: please take care of the formatting

**Review by dwivedianurag — APPROVED** [review]
please do the needful before merging the pr

File: docs/demos/cf-lbrands.md (hunk 1 above)
Comment (tushar-macrometa) [reply docs/demos/cf-lbrands.md:7]: @dwivedianurag What formatting issue?

Comment (dwivedianurag) [reply docs/demos/cf-lbrands.md:7]: @tushar-macrometa it is as expected

**Review by elof — APPROVED** [review]

**Pull request merged**